integration_test/third_party_apps_data/applications/hadoop/centos_rhel/install

```diff
@@ -0,0 +1,144 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir -p \
+    /opt/hadoop/logs \
+    /opt/volume/datanode \
+    /opt/volume/namenode
+
+wget https://dlcdn.apache.org/hadoop/common/hadoop-3.3.1/hadoop-3.3.1.tar.gz
+sudo tar -xf hadoop-3.3.1.tar.gz -C /opt/hadoop --strip 1
+
+JAVA_HOME="$(dirname "$(dirname "$(readlink "$(readlink "$(which javac)")")")")"
+
+cat <<EOF | sudo tee /opt/hadoop/env
```

Comment:
Use an `EnvironmentFile` in the systemd unit so this isn't duplicated so many times?

Reply:
We can't use an environment file because the environment variables are used in three spots `systemd unit`, `core-site.xml`, and `hadoop users bashrc` and the environement file, I don't believe, allows the export for the environment file.

Reply:
I think you could put the export statement on a separate line e.g.
```
A=1
B=2
export A B
```
and systemd would ignore the export line. I haven't tested it though.

Reply:
Haven't been able to apply this successfully. I can't seem to find anything online as an example of anyone doing this either. Seems that once we add in the ability to refactor out this into a separate common script this will be a lot cleaner anyways as a side effect, if that was your concern.